source/_docs/installation/docker.markdown

```diff
@@ -267,3 +267,7 @@ or in a `docker-compose.yml` file:
       restart: always
       network_mode: host
 ```
+
+<div class='note'>
+On Mac, USB devices are <a href="https://github.com/docker/for-mac/issues/900">not passed through</a> by default. Follow the instructions in <a href="https://dev.to/rubberduck/using-usb-with-docker-for-mac-3fdd">Using USB with Docker for Mac</a> by Christopher McClellan if your device is not showing up.
```

Comment:
Please use common Markdown for creating links.

Reply:
@frenck Good point. Should I change the last paragraph of `Synology NAS` as well? I copied mine from there. What's important to note: In order for this to work, we'll need an empty line after `<div class='note'>` and before `</div>`.

Reply:
Yes, it needs an empty line to separate HTML from Markdown, as is documented in our documentation standards.

Improvements to other parts of the documentation is, of course, always welcome.

Reply:
Whoups, sorry, RTFM. :)
Fixed.